src/server.c

```diff
@@ -5919,14 +5920,15 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
                 }
                 const char *state = replstateToString(slave->replstate);
                 if (state[0] == '\0') continue;
+                offset_lag = server.master_repl_offset - slave->repl_ack_off;
                 if (slave->replstate == SLAVE_STATE_ONLINE)
                     lag = time(NULL) - slave->repl_ack_time;
 
                 info = sdscatprintf(info,
                     "slave%d:ip=%s,port=%d,state=%s,"
-                    "offset=%lld,lag=%ld\r\n",
+                    "offset=%lld,offset_lag=%lld,lag=%ld\r\n",
```

Comment:
I don't really like the name, but I can't think of a better one. I would prefer something like "relative_offset" maybe?